cfgov/ask_cfpb/models/pages.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.utils import timezone
+from django.db import models
+
+from wagtail.wagtailadmin.edit_handlers import (
+    FieldPanel,
+    ObjectList,
+    TabbedInterface)
+from wagtail.wagtailcore.fields import RichTextField
+from wagtail.wagtailcore.models import Page, PageManager
+from wagtail.wagtailsearch import index
+
+from v1.models import CFGOVPage
+from ask_cfpb.models import Answer
+
+
+class AnswerPage(CFGOVPage):
+    """
+    Page type for Ask CFPB answers.
+    """
+    question = RichTextField(blank=True, editable=False)
+    answer = RichTextField(blank=True, editable=False)
+    snippet = RichTextField(
+        blank=True, help_text='Optional answer intro', editable=False)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    publish_date = models.DateTimeField(default=timezone.now)
+    answer_base = models.ForeignKey(
+        Answer,
+        blank=True,
+        null=True,
+        on_delete=models.PROTECT)
```

Comment:
It might be nice to have a `related_name='answer_pages'` here so that you can do `answer.answer_pages` instead of needing `answer.answerpage_set`.